packages/server/src/rpc.ts

```diff
@@ -25,31 +40,36 @@ export function rpcHandler(
       if (typeof req.body.params === 'undefined') {
         const error = {message: 'Request body is missing the `params` key'}
         log.error(`${logPrefix} failed: ${JSON.stringify(error)}\n`)
-        return res.status(400).json({
+        res.status(400).json({
           result: null,
           error,
         })
+        return next()
```

Comment:
Should these `next()` calls contain the `{result, error}` parameter in case the RPC has custom middleware that needs to react to the result of the RPC call?

Reply:
Also on this, did you see [the docs](https://blitzjs-com-git-middleware.blitzjs.vercel.app/docs/middleware)?

`result` is available via `res.blitzResult`